tests/test_mpi.py

```diff
@@ -1445,16 +1445,43 @@ def test_staggering(self):
         op(time_M=2)
 
         # Expected norms computed "manually" from sequential runs
-        assert np.isclose(norm(ux), 5408.574, rtol=1.e-4)
-        assert np.isclose(norm(uxx), 60904.192, rtol=1.e-4)
-        assert np.isclose(norm(uxy), 58555.359, rtol=1.e-4)
+        assert np.isclose(norm(ux), 6253.4349, rtol=1.e-4)
+        assert np.isclose(norm(uxx), 80001.0304, rtol=1.e-4)
+        assert np.isclose(norm(uxy), 61427.853, rtol=1.e-4)
 
 
 class TestIsotropicAcoustic(object):
 
     """
     Test the isotropic acoustic wave equation with MPI.
     """
+    _shapes = {1: (60,), 2: (60, 70), 3: (60, 70, 80)}
+    _so = {1: 12, 2: 8, 3: 4}
+
+    def gen_serial_norms(self, nd):
```

Comment:
use `memoized_meth` to avoid recomputing this

Reply:
It's only called once